python/tskit/__init__.py

```diff
@@ -39,6 +39,14 @@
 #: decreasing genomic coordinate values).
 REVERSE = _tskit.REVERSE
 
+#: The allele mapping where the strings "0" and "1" map to genotype
+#: values 0 and 1.
+ALLELES_01 = ("0", "1")
+
+#: The allele mapping where the four nucleotides A, C, G and T map to
+#: the genotype integers 0, 1, 2, and 3, respectively.
+ALLELES_ACGT = ("A", "C", "G", "T")
```

Comment:
Do we also want to support `ALLELES_acgt` (lowercase). Probably not, as it's trivial to do so yourself, and uppercase is the norm (lowercase tends to be used for soft masked regions, although I don't really know the conventions well).

Reply:
We can add it easily if someone wants it.